database/seed.js

```diff
@@ -0,0 +1,38 @@
+const faker = require('faker');
+const db = require('../database/model.js');
+const dataFormat = require('../helpers/dataFormat.js');
+
+
+const populate = function() {
+
+  // db.dropDatabase();
+
+  const dummyData = [];
+
+  const neighborhoods = ['Last Hearth', 'Crofter\'s Village', 'Hornwood', 'Wolfswood', 'Cerwyn'];
+
+  const cities = ['Ramsgate', 'Dreadfort', 'Winterfell', 'White Harbor', 'Barrowton'];
+
+    for (let i = 0; i < 100; i++) {
+
+      const fakeHome = {
+        property_id: '',
```

Comment:
what's this property_id property for?

Reply:
It was for an api call that I'm no longer using. I went ahead and removed it